libraries/crypto/Cargo.toml

```diff
@@ -17,10 +17,8 @@ byteorder = { version = "1", default-features = false }
 hex = { version = "0.3.2", default-features = false, optional = true }
 ring = { version = "0.16.11", optional = true }
 untrusted = { version = "0.7.0", optional = true }
-serde = { version = "1.0", optional = true, features = ["derive"] }
-serde_json = { version = "=1.0.69", optional = true }
 regex = { version = "1", optional = true }
 
 [features]
-std = ["hex", "ring", "rng256/std", "untrusted", "serde", "serde_json", "regex"]
```

Comment:
I think this file was correct. We use `serde_json` for tests.

Reply:
Ah, I see. In that case, I think I had random issues with the serde version before I added it to the main `Cargo.toml` as well. Not sure how to reproduce, so maybe we can just remove it outside of crypto and try again until that happens.

Reply:
Ah, I see. In that case, I think I had random issues with the serde version before I added it to the main `Cargo.toml` as well. Not sure how to reproduce, so maybe we can just remove it outside of crypto and try again until that happens.

Reply:
Yes, now the change is making sense.

I think it was hard to reproduce because of some many states (the `Cargo.lock` file**s** in multiple directories). The best way to test is `./reset.sh` (committing code first otherwise it's lost) but it can be a bit slow for fast iteration.